src/package/init.ts

```diff
@@ -109,8 +121,8 @@ export function init (modules: Array<Partial<Module>>, domApi?: DOMAPI) {
       const dot = dotIdx > 0 ? dotIdx : sel.length
       const tag = hashIdx !== -1 || dotIdx !== -1 ? sel.slice(0, Math.min(hash, dot)) : sel
       const elm = vnode.elm = isDef(data) && isDef(i = data.ns)
-        ? api.createElementNS(i, tag)
-        : api.createElement(tag)
+        ? api.createElementNS(i, tag, vnode.data)
+        : api.createElement(tag, vnode.data)
```

Comment:
Here you can just use `data` as defined above instead of `vnode.data`.

Reply:
these nested ternaries are very hard for humans to read. At least, this human struggles with reading them. :)